exercises/robot-simulator/src/test/java/RobotTest.java

```diff
@@ -6,20 +6,24 @@
 public class RobotTest {
 
     @Test
-    public void testRobotIsCreatedWithCorrectInitialPosition() {
+    public void testRobotIsCreatedWithCorrectInitialPositionAndOrientation() {
+        final Orientation initialOrientation = Orientation.NORTH;
         final GridPosition initialGridPosition = new GridPosition(0, 0);
         final Robot robot = new Robot(initialGridPosition, Orientation.NORTH);
```

Comment:
Sorry, one final thing, to be more consisten with the existing code please use the intialGridOrientation when creating the robot:
```
final Robot robot = new Robot(initialGridPosition, initialOrientation);
```
🙂